documentation/docs/admin/production-setup/multiple-artemis-instances.mdx

```diff
@@ -0,0 +1,745 @@
+
+You can verify that a build agent has been successfully added to the cluster by checking the Build Agent View in the Server Administration. It may take a few seconds for the build agent to show up:
+
+<Image src={buildAgentViewImg} size={ImageSize.medium} />
+
+
+## Running multiple instances locally
+
+For testing purposes, you can also run multiple instances on the same machine. You can do this by using
+different ports and a unique instance ID for each instance.
+
+1. In `application-local.yml`, add the following configuration:
+
+    ```yaml
+    eureka:
+        client:
+            enabled: true
+    ```
+
+2. Create additional run configurations for each instance. You will have to add CLI arguments to each additional run
+    configuration to set the instance ID and the port, e.g. ``--server.port=8081 --eureka.instance.instanceId="Artemis:2"``.
+    Also, make sure that only one instance has the ``scheduling`` profile enabled:
+
+    <Image src={runConfigImg} size={ImageSize.medium} />
+
+
+3. Start the registry service, e.g., by running `docker compose -f docker/broker-registry.yml up`.
+
+4. Start the first instance with the default run configuration (no additional CLI arguments, `scheduling` enabled)
+    and wait until it is up and running.
+
+5. Start the remaining instances.
+
+You should now be able to see all instances in the registry interface at `http://localhost:8761`.
+
+
+### Running multiple instances locally with Docker
+
+You can also run multiple instances of Artemis locally using Docker. This will start 3 Artemis instances, each running
+on a its own container. A load balancer (nginx) will be used to distribute the requests to the different instances. The
+load balancer will be running in a separate container and will be accessible on ports 80/443 of the host system. The
+instances will be registered in the registry service running on a separate container. The instances will use the registry
+service to discover each other and form a Hazelcast cluster. Further details can be found in :ref:`Database Cache`. The
+instances will also use a ActiveMQ Artemis broker to synchronize WebSocket messages. Further details can be found in [WebSockets](#websockets). In summary, the setup will look like this:
+
+* 3 Artemis instances:
+    * artemis-app-node-1: using following spring profile: ``prod,localvc,localci,core,scheduling,docker``
+    * artemis-app-node-2: using following profile: ``prod,localvc,localci,buildagent,core,docker``
+    * artemis-app-node-3: using following profile: ``prod,buildagent``
+* A MySQL database addressable on port 3306 of the host system
+* A Load balancer (nginx) addressable on ports 80/443 of the host system: ``http(s)://localhost``
+* A Registry service addressable on port 8761 of the host system: ``http://localhost:8761``
+* An ActiveMQ broker
+
+
+<Image src={multiNodeSetupImg} size={ImageSize.medium} />
```

Comment:
Can we also make the pictures bigger? For me they seem quite small

<img width="945" height="575" alt="image" src="https://github.com/user-attachments/assets/0fa78dc2-deed-4c1f-8351-4eb24ba51bcb" />


<img width="838" height="469" alt="image" src="https://github.com/user-attachments/assets/5c4cbf96-ad04-4e48-b4cf-cc4e627e1480" />